packages/sfui/frameworks/vue/components/SfIcons/SfIconAlokaiFull.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <SfIconBase :size="size" viewBox="0 0 129 28" data-testid="alokai-full"
+    ><path
+      d="m21.427 9.333 3.248.013A3.326 3.326 0 0 0 28 6.021V3.325A3.326 3.326 0 0 0 24.675 0h-2.696a3.326 3.326 0 0 0-3.325 3.325V6.56a2.764 2.764 0 0 1-2.76 2.76H11.54a2.22 2.22 0 0 0-2.22 2.221v4.353a2.764 2.764 0 0 1-2.76 2.76H1.103c-.616 0-1.104.5-1.104 1.104v7.125c0 .616.5 1.104 1.104 1.104H8.23c.616 0 1.104-.5 1.104-1.104v-5.456a2.764 2.764 0 0 1 2.76-2.76h3.8a2.764 2.764 0 0 1 2.76 2.76V28h9.334v-9.333h-6.573a2.764 2.764 0 0 1-2.76-2.76v-3.8c.013-1.528 1.258-2.774 2.773-2.774Zm-16.754.013a4.673 4.673 0 1 0 0-9.346 4.673 4.673 0 0 0 0 9.346ZM65.012 3.12h2.555v21.76h-2.555V3.12Zm63.023 6.213h-2.555V24.88h2.555V9.333Z" /><path
+      fill-rule="evenodd"
```

Comment:
Question: if fill-rule required here (and below as well)?

Reply:
And in another file

Reply:
I assumed it is but I will check it

Reply:
https://github.com/vuestorefront/storefront-ui/pull/3070/commits/471c59ce9d72ee9b528c6db56a37eac86397f4a1

Reply:
noice!